This will expand to
| #define ICONV_CONST <tabs> AM_ICONV */
otherwise.

--- lynx2-8-7.orig/config.hin	2007-08-03 01:24:27.000000000 +0200
+++ lynx2-8-7/config.hin	2009-06-13 11:46:29.000000000 +0200
@@ -176,7 +176,7 @@
 #undef HAVE___ARGZ_COUNT	/* AM_GNU_GETTEXT */
 #undef HAVE___ARGZ_NEXT		/* AM_GNU_GETTEXT */
 #undef HAVE___ARGZ_STRINGIFY	/* AM_GNU_GETTEXT */
-#undef ICONV_CONST		/* AM_ICONV */
+#undef ICONV_CONST
 #undef IGNORE_CTRL_C		/* FIXME: make tests? */
 #undef INCLUDE_PROTOTYPES	/* CF_SOCKS5 */
 #undef INSTALL_ARGS		/* CF_PATH_PROG(install) */
